ext/handlers/filters/message/message.go

```diff
@@ -47,6 +47,16 @@ func ForwardFromChatID(id int64) filters.Message {
 	}
 }
 
+func Regex(pattern string) filters.Message {
+	return func(m *gotgbot.Message) bool {
+		matched, err := regexp.MatchString(pattern, m.Text)
```

Comment:
Regexes that are reused should be compiled for speed, see `regex.Compile`

Reply:
Agree

Reply:
Can you please explain a bit more, please?